inst/tests/testthat/test-helloworld.R

```diff
@@ -0,0 +1,34 @@
+library(testthat)
+library(methods)
+library(grpc)
+library(processx)
+
+context('helloworld example')
+
+## start a server in the background
+p <- process$new(commandline = "R -e \"demo('helloserver', 'grpc', ask = FALSE)\"", stdout = "|", stderr = "|")
```

Comment:
Not familiar with processx, what's the benefit over just using `system2()` appropriately?

Reply:
You can monitor and interact with the process in the background, eg read stdout from time to time, and kill it later.